### PR TITLE
Added question into the Article #787 with answer.

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving Issue #787 with an answer. 

@nenonix  @JohanFreelancer9  @scanum, please kindly proceed to the Copy review! and thank you in advance.

### CODE REVIEW

File: SecurityCompliance/search-cloud-based-mailboxes-for-on-premises-users.md [hunk 1]
```diff
@@ -155,3 +155,7 @@ No.
  **Can Content Search find older Teams chats for on-premises users before the time my organization submitted the request to enable this feature?**
   
 Microsoft started storing the Teams chat data for on-premises users on January 31, 2018. So, if the identity of an on-premises Teams user has been synched between Active Directory and Azure Active Directory since this date, then their Teams chat data will be stored in a cloud-based mailbox and will be searchable using Content Search. Microsoft is also working on storing Teams chat data from prior to January 31, 2018 in the cloud-based mailboxes for on-premises users. More information about this will be available soon.
+
+ **Does a Teams and Exchange Online licesse need to be assigned to the user to create the mailbox for online storage?**
```
Comment (JohanFreelancer9) [inline SecurityCompliance/search-cloud-based-mailboxes-for-on-premises-users.md]:
```suggestion
 **Does a Teams and Exchange Online license need to be assigned to the user to create a mailbox for online storage?**
```

File: SecurityCompliance/search-cloud-based-mailboxes-for-on-premises-users.md [hunk 2]
```diff
@@ -155,3 +155,7 @@ No.
...
+  
+You need at least a license of exchange online plan 1 to get data to save data into the mailbox, relative to the  Microsoft Teams License it's required to access the service. In concrete, yes You'd need that the user has an MS Teams license and a mailbox license at least to take advantage of this feature.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/search-cloud-based-mailboxes-for-on-premises-users.md]:
```suggestion
You need at least a license for Exchange Online Plan 1 to to get data to save into the mailbox, in addition to the  Microsoft Teams License that is required to access the service. So effectively yes, the user does require at least an MS Teams license and a mailbox license to take advantage of this feature.
```

**Comment by j0rt3g4** [comment]
@JohanFreelancer9  Thank you!

**Comment by dariomws306** [comment]
Sorry guys but this PR doesn't look good and it's published.

image

Question is not in bold.

**Comment by markjjo** [comment]
@dariomws306 Thanks for heads up; fix is merged to master: https://review.docs.microsoft.com/en-us/Office365/SecurityCompliance/search-cloud-based-mailboxes-for-on-premises-users?branch=master#frequently-asked-questions

**Comment by dariomws306** [comment]
Thank you @markjjo!

**Pull request merged**